packages/cli/src/commands/site/deploy/index.ts

```diff
@@ -117,7 +117,13 @@ const command: CommandModule<
   },
   handler: async (args) => {
     const command = await import("./siteDeployCommand.mjs");
-    await command.default(args);
+    const selectedVersion = args.version != null
+      ? args.version
+      : {
+        type: args.autoVersion as "git-describe",
```

Comment:
Can we similarly make this type more specific and/or use coerce similar to third party app rid?